compiler/circle-opselector-test/README.md

```diff
@@ -0,0 +1,52 @@
+# Circle-OpSelector-test
+
+## Usage
+
+- This test didn't consider **if** and **while** nodes yet. 
+
+**Single file test**
+
+- `tflite-file-name` must be contained in [tflite receipes](https://github.com/Samsung/ONE/tree/master/res/TensorFlowLiteRecipes)
+- `selected nodes` format is `"0 1 3"`
+
+```
+$ ./opselector_test.sh [tflite-file-name] [selected nodes]
+# ex)
+$ ./compiler/circle-opselector-test/opselector_test.sh Part_Sqrt_Rsqrt_Add_002 "0 1 2 3"
+```
+
+This test compares between the result of `circle-opselector` and [select-operator.py](https://github.com/Samsung/ONE/blob/master/tools/tflitefile_tool/select_operator.py) in tflite_tool.
+
+The detailed process is as follows
+
+1. Prepare a `tflite` model.
+2. Create a `circle` version of that `tflite` model using 'tflite2circle`.
+3. Create a `tflite` subgraph using `select_operator.py`.
```

Comment:
I think there's no need to use this tool. If we find some defects of `select_operator.py` in the future, `circle-opselector` should be fixed as well.

You need just one recipe that is going to be input of `circle-opselector` and another recipe that is going to be expected output of `circle-opselector`.

Reply:
+1 for @mhs4670go. 

You may use `select_operator.py` to get correct answer during early develop stage, but this tool doesn't need to (and should not) depend on `seleclt_operator.py`.

Reply:
I absolutely agree this comment. I will gather test data set(before select, after select) using recipe instead of using `select_operator.py`